src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -376,6 +355,8 @@ void VfsCfApi::scheduleHydrationJob(const QString &requestId, const QString &fol
     job->setJournal(params().journal);
     job->setRequestId(requestId);
     job->setFolderPath(folderPath);
+    job->setIsEncryptedFile(!record._e2eMangledName.isEmpty());
```

Comment:
better use _isE2eEncrypted attribute (and if needed add a getter for it)

Reply:
> better use _isE2eEncrypted attribute (and if needed add a getter for it)

@mgallien So, `_isE2eEncrypted  `needs to be in `private:` and a getter must be used everywhere in code? I am just not seeing a huge benefit of a getter for a public member.

Reply:
The following reason I see that speak for adding getters for public members:

- You leak implementation details out of your class. If you need in the future to change the logic that yields the value `_e2eMangledName` it will be difficult if it is just a member variable. If it is a method, the change will be easier.
- At some point you may want to change the logic how `_e2eMangledName` gets set and you want to be sure that no one can bypass the logic. If you have `_e2eMangledName` a public member that will be difficult.

Reply:
@FlexW Well, I could agree with that. But, then, we must turn `_e2eMangledName` into private and have a setter and getter for it. Otherwise, this improvement is incomplete and does not prevent other code to get and set `_e2eMangledName` directly?

Reply:
Yes exactly :)